src/core_plugins/elasticsearch/index.js

```diff
@@ -15,8 +16,6 @@ const DEFAULT_REQUEST_HEADERS = [ 'authorization' ];
 
 module.exports = function ({ Plugin }) {
   return new Plugin({
-    require: ['kibana'],
```

Comment:
It does feel a bit odd to me to have it under the ui folder when it is only being used server side. Did @spalger's example of consuming the uiExported mappings in the ES plugin's init function work? uiExports are still slightly mysterious to me so I'll defer to his judgement.